Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -264,38 +265,26 @@ void Calendar::update_tiles(unsigned view_year, unsigned view_month)
 {
     set_view_date(view_year, view_month);
 
-    unsigned months;
-    mode() == Month ? months = 1 : months = 12;
+    auto now = Core::DateTime::now();
+    unsigned months = mode() == Month ? 1 : 12;
     for (unsigned i = 0; i < months; i++) {
         if (mode() == Year)
             view_month = i + 1;
+
+        unsigned first_day_of_week_for_month = ::day_of_week(view_year, view_month, 1);
```

Comment:
No need to scope AK here

Reply:
I'm trying to reduce the amount of DateTime instantiations as much as possible. If `::day_of_week` isn't something we want to use directly, would you prefer this to still avoid a new instantiation?

```cpp
unsigned first_day_of_week_for_month = (first_day_of_previous_month.weekday() + first_day_of_previous_month.days_in_month()) % 7;
```

Reply:
I mean you can remove the `::`
AK is global and no ambiguity here

Reply:
Resolved!